lab_6_pipeline/pipeline.py

```diff
@@ -52,11 +117,20 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        for article in self._corpus.get_articles().values():
+            from_meta(article.get_meta_file_path(), article)
```

Comment:
why do we need it?

Reply:
I thought we should have all information about the article in the Article object.

Reply:
I found the place to add meta in documentation, so deleted this line